report_account_move_settlement/reports/report_settlement_templates.xml

```diff
@@ -53,13 +53,28 @@
                                 t-field="o.partner_id"
                                 t-options='{"widget": "contact", "fields": ["name"], "no_marker": True}'
                             />
+                        <div t-if="o.partner_id.vat" class="mb-2">
+                            <span>登録番号:</span>
+                            <span
+                                    t-field="o.partner_id.vat"
+                                    t-options='{"no_marker": True}'
```

Comment:
`t-opitons='{"no_marker": True}'`  

Is this option necessary?

Reply:
I modeled the format after the pre-existing code and added lines 60 and 75.
I'll do a test first and if it's ok, I'll remove those 2 lines of code.

Reply:
@liuhehe1995 @kanda999 
We should add `t-options='{"no_marker": True}'` for reports because we don't need to automatically update `o.partner_id.vat` in real time. It can reduce the overhead of monitoring changes for `vat` field.